### PR TITLE
Fix reverse translate issue

### PR DESCRIPTION
**Pull request by richardwxn**
Fix for https://github.com/istio/istio/issues/18289
Also as a TODO, create https://github.com/istio/istio/issues/18392 to track update of the reverse translation table

### CODE REVIEW

File: cmd/mesh/testdata/manifest-generate/output/pilot_override_kubernetes.yaml [hunk 1]
```diff
@@ -18,6 +18,29 @@
 
 # NodeAgent component is disabled.
 
+apiVersion: v1
```
Comment (ostromart) [inline cmd/mesh/testdata/manifest-generate/output/pilot_override_kubernetes.yaml]: why are there such huge diffs here? it seems like some resources are reordered.

Comment (richardwxn) [reply cmd/mesh/testdata/manifest-generate/output/pilot_override_kubernetes.yaml]: yep I think it doesn't makes sense, but as I said here: https://github.com/istio/operator/pull/482#pullrequestreview-308998258, seems like the output order is not stable

File: pkg/translate/translate_value.go [hunk 2]
```diff
@@ -532,10 +532,17 @@ func (t *ReverseTranslator) isEnablementPath(path util.Path) bool {
 	if len(path) < 2 || path[len(path)-1] != "enabled" {
 		return false
 	}
+
+	pstr := path.String()
+	if pstr == "mixer.policy.enabled" || pstr == "mixer.telemetry.enebaled" ||
```
[the remark this answers is not in the document]
Comment (richardwxn) [reply pkg/translate/translate_value.go]: done

**Comment by richardwxn** [comment]
This one needs to get in soon as it fixes the breaking migration tools, can you check again plz @ostromart

**Review by ostromart — APPROVED** [review]

**Pull request merged**